app/mm-redux/actions/threads.ts

```diff
@@ -0,0 +1,242 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {ThreadTypes, PostTypes, UserTypes} from '@mm-redux/action_types';
```

Comment:
think of moving this to `actions/views/threads.ts` probably you don't need to add anything under `mm-redux`